### PR TITLE
Issue fixes

### PR DESCRIPTION
**Pull request by mcyprian**
In this PR
- 3 issues fixed
- webtest marker
- update of DandifiedNameConvertor test suite

### CODE REVIEW

File: pyp2rpm/extract_distribution.py [hunk 1]
```diff
@@ -65,7 +65,10 @@ def _list(var):
         if isinstance(var, str):
             var = var.split('\n')
         elif not isinstance(var, list):
-            raise DistutilsOptionError("{} is not a list".format(var))
+            try:
+                var = list(var)
+            except TypeError:
+                raise DistutilsOptionError("{} can not be converted to list".format(var))
```
Comment (hroncok) [inline pyp2rpm/extract_distribution.py]: http://www.oxforddictionaries.com/words/cannot-or-can-not

File: tests/test_extract_distribution.py [hunk 2]
```diff
@@ -9,6 +9,8 @@ class TestExtractDistribution(object):
         (['pkg'], ['pkg']),
         (None, []),
         ('pkg >= 2.5\npkg2', ['pkg >= 2.5', 'pkg2']),
+        (('pkg'), ['pkg']),
+        (('pkg',), ['pkg']),
```
Comment (hroncok) [inline tests/test_extract_distribution.py:13]: put a generator in there as well? such as:

``` python
(p for p in ('pkg',))
```

**Comment by hroncok** [comment]
Otherwise looking good.

**Comment by mcyprian** [comment]
Fixed.

**Pull request merged**